documentation/observability/5-deploy-to-kyma.md

```diff
@@ -40,51 +35,57 @@ To create the SAP Cloud Logging service instance, follow these steps:
     kind: ServiceBinding
     metadata:
       name: cls-binding
     spec:
-      serviceInstanceName: created-with-sap-btp-service-operators
+      serviceInstanceName: sap-cloud-logging
       externalName: cloud-logging-created-with-sap-btp-service-operators
       secretName: sap-cloud-logging
       credentialsRotationPolicy:
         enabled: true
-        rotationFrequency: 168h
+        rotationFrequency: 1440h
 
     ```
     To apply the configuration, use the following command:
     ```
-    kubectl apply -n sap-cloud-logging-integration -f clsbinding.yaml
+    kubectl apply -n sap-cloud-logging-integration -f cloud-logging-binding.yaml
     ```
     
-7. Ship the logs to the SAP Cloud Logging service. See [Ship Logs to SAP Cloud Logging](https://kyma-project.io/#/telemetry-manager/user/integration/sap-cloud-logging/README?id=ship-logs-to-sap-cloud-logging).
+1. Ship the logs to the SAP Cloud Logging service. See [Ship Logs to SAP Cloud Logging](https://help.sap.com/docs/btp/sap-business-technology-platform/integrate-with-sap-cloud-logging?ship-logs-to-sap-cloud-logging).
 
-8. Ship the distributed traces to the SAP Cloud Logging service. See [Ship Distributed Traces to SAP Cloud Logging](https://kyma-project.io/#/telemetry-manager/user/integration/sap-cloud-logging/README?id=ship-distributed-traces-to-sap-cloud-logging).
+1. Ship the distributed traces to the SAP Cloud Logging service. See [Ship Distributed Traces to SAP Cloud Logging](https://help.sap.com/docs/btp/sap-business-technology-platform/integrate-with-sap-cloud-logging?ship-distributed-traces-to-sap-cloud-logging).
 
-9. Ship the metrics to the SAP Cloud Logging service. See [Ship Metrics to SAP Cloud Logging](https://kyma-project.io/#/telemetry-manager/user/integration/sap-cloud-logging/README?id=ship-metrics-to-sap-cloud-logging).
+1. Ship the metrics to the SAP Cloud Logging service. See [Ship Metrics to SAP Cloud Logging](https://help.sap.com/docs/btp/sap-business-technology-platform/integrate-with-sap-cloud-logging?ship-metrics-to-sap-cloud-logging).
 
     Once the configurations are applied, you can check the status of all telemetry configurations in the Kyma dashboard. They should look like this: 
     <img src="./images/pipelines.png" />
 
     > [!TIP]
     > Learn how to access [Kyma dashboard](https://learning.sap.com/learning-journeys/deliver-side-by-side-extensibility-based-on-sap-btp-kyma-runtime/using-the-kyma-dashboard_d23b12a1-d17c-491d-a80b-cb78039e317e).
 
-10.  [Kyma Dashboard Integration](https://kyma-project.io/#/telemetry-manager/user/integration/sap-cloud-logging/README?id=ship-metrics-to-sap-cloud-logging).
+1.  For easier access, adjust the Kyma dashboard navigation. See [Kyma Dashboard Integration](https://help.sap.com/docs/btp/sap-business-technology-platform/integrate-with-sap-cloud-logging?locale=en-US&version=Cloud#set-up-kyma-dashboard-integration).
 
 # Deploy the Incident Management Application in the SAP BTP, Kyma Runtime
 
```

Comment:
Line 95: Why is this just a lonely link as part of a list with 1 item? Could this be a sentence similar to line 92 usw?

Reply:
Made a proposal

Reply:
LGTM